src/domains/vote/components/DelegateTable/DelegateTable.tsx

```diff
@@ -164,6 +164,7 @@ export const DelegateTable = ({
 
 	const handleSelectPage = (page: number) => {
 		setCurrentPage(page);
+		window.scrollTo({ top: 0, behavior: "smooth" });
```

Comment:
in `News.tsx` we have a `useEffect(() => window.scrollTo({ top: 0, behavior: "smooth" }), [currentPage, coins]);`, would something like that work for delegates too? e.g. watching a change in `page` prop and then scroll up instead of adding it to this method

Reply:
Sure, we could put it inside useEffect. Are there other actions that require to scroll to the top?

Reply:
only when the page changes I think as that forces new content in the table 🤔